index.json

```diff
@@ -2,5 +2,7 @@
   "aws/monolith",
   "aws/microservice-ecommerce",
   "aws/datalake",
-  "docker/simple-demo-app"
+  "docker/simple-demo-app",
+  "GlassFish",
```

Comment:
I suggest to rename this to `glassfish-demo` and move it under `server` so it would be `server/glassfish-demo` here